ScoutSuite/providers/gcp/provider.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, project_id=None, folder_id=None, organization_id=None,
 
         super(GCPProvider, self).__init__(report_dir, timestamp, services, skipped_services, thread_config)
 
-    def authenticate(self, key_file=None, user_account=None, service_account=None, **kargs):
+    def authenticate(self, key_file=None, user_account=None, service_account=None, all=False, **kargs):
```

Comment:
I don't think this is right, `all` should be available in `self`, as for `project_id`/`folder_id`/etc

Reply:
Moved to GCPProvider object and made `gcp-scan-all` available as kwargs